app/main/presenters/service_presenters.py

```diff
@@ -24,8 +24,8 @@ def _add_as_attribute_if_key_exists(self, key, service_data):
         if key[0] in service_data:
             setattr(self, key[1], service_data[key[0]])
 
-    def __init__(self, service_data, service_questions, lots_by_slug):
-        self.service_questions = service_questions
+    def __init__(self, service_data, builder, lots_by_slug):
+        self.summary_manifest = builder.summary(service_data)
```

Comment:
@TheDoubleK and I don't really like the word 'manifest' either, as 'manifest' is really the _file_, that the (whatever) class uses to assemble the questions together.

Given the name of the class is actually manifest, then probably you're right and we should stick to that, but I'm sorely tempted to talk to @allait about renaming it, because I think calling it 'manifest' is actually really confusing. Really, it's a `Form` - I think :-)